schema/mysql/upgrades/dependencies2.sql

```diff
@@ -0,0 +1,2 @@
+alter table redundancy_group_state
+    add last_state_change bigint unsigned not null;
```

Comment:
https://github.com/Icinga/icingadb/blob/dbd05f20420ed986f8d36942ca6b3582cbc8d472/schema/mysql/upgrades/dependencies.sql#L28

Unless I am missing something, it is already there.